afl-fuzz.c

```diff
@@ -3543,10 +3553,16 @@ static void maybe_update_plot_file(double bitmap_cvg, double eps) {
 
   u32 singletons = 0;
   u32 doubletons = 0;
+  u32 tripletons = 0;
+  u32 quadrupletons = 0;
+  u32 quintupletons = 0;
   struct queue_entry* q = queue;
   while (q) {
     if (q->n_fuzz == 1) singletons ++;
     if (q->n_fuzz == 2) doubletons ++;
+    if (q->n_fuzz == 3) tripletons ++;
+    if (q->n_fuzz == 4) quadrupletons ++;
+    if (q->n_fuzz == 5) quintupletons ++;
```

Comment:
If you have multiple "x-tons", can you create an array representing them? This may be more efficient.
```C
u32 x_tons[8] = {};
struct queue_entry* q = queue;
while (q) {
  if (q->n_fuzz <= 8 && q->n_fuzz != 0)
    x_tons[q->n_fuzz - 1] ++;
  q = q->next;
}
```
Now, singletons are at `x_tons[0]`, doubletons are at `x_tons[1]`, etc.